databases/week2/hyf_lesson2.sql

```diff
@@ -0,0 +1,59 @@
+-- =============================================================================
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+-- =============================================================================
+
+SELECT user.id, task.title, user.email
+FROM user, task, user_task
+WHERE user.email LIKE "%@spotify.com"
+AND user.id = user_task.user_id
+AND user_task.task_id = task.id;
+
+
+
+-- =============================================================================
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+-- =============================================================================
+
+SELECT user.name AS user_name, task.title AS task_name, status.name as status_name
+FROM user, task, status, user_task
+WHERE user.id = user_task.user_id
+AND user_task.task_id = task.id
+AND task.status_id = status.id
+AND user.name LIKE "Donald Duck" 
+AND status.name LIKE "Not started";
+
+SELECT user.name AS user_name, task.title AS task_name, status.name as status_name
+FROM user_task
+JOIN user ON user_id = user.id
+JOIN task ON task.id = user_task.task_id
+JOIN status ON status.id = status_id
+WHERE user.name = "Donald Duck"
+AND status.name = "Not started";
+
+
+
+
+-- =============================================================================
+--  Get all the tasks for 'Maryrose Meadows' that were created in september 
+--  (hint: month(created)=month_number)
+-- =============================================================================
+
+SELECT task.title as task_title, task.id as task_id, user.name as user_name, task.created
+FROM task, user_task, user
+WHERE user.name LIKE "Maryrose Meadows"
+AND user.id = user_task.user_id
+AND user_task.task_id = task.id
+AND month(created) = 09;
+
```

Comment:
This is your code result
![image](https://user-images.githubusercontent.com/84173609/136687427-9119da0f-c2fe-415a-b435-63f6799e10fd.png)

And this is mine
![image](https://user-images.githubusercontent.com/84173609/136687436-f8943736-1b2d-494d-b70a-cf8574d53a74.png)

Now I am confused which result is right.
I can try to use JOIN on my code instead of all long line that I used

Reply:
The same here as above.